app/Http/Requests/DeleteUserRequest.php

```diff
@@ -41,30 +35,41 @@ public function rules(): array
                 'assigned_assets' => $user_to_delete->assets()->count(),
                 'assigned_licenses' => $user_to_delete->licenses()->count(),
                 'assigned_accessories' => $user_to_delete->accessories()->count(),
+                'deleted_at' => $user_to_delete->deleted_at,
             ]);
         }
+    }
 
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
         return [
-            'id' => ['exists:users,id'],
-            'user' =>  Rule::notIn([Auth::user()->id]),
+            'user' =>  ['in:null|not_in:auth()->user()->id'],
             'managed_users' =>  Rule::in([0]),
             'managed_locations' => Rule::in([0]),
             'assigned_assets' => Rule::in([0]),
             'assigned_licenses' => Rule::in([0]),
             'assigned_accessories' => Rule::in([0]),
+            'deleted_at' => Rule::in([null]),
         ];
     }
 
     public function messages(): array
     {
 
-        $user_to_delete = User::find(request()->route('user'));
-        $messages = ['id.exists' => trans('admin/users/message.user_not_found')];
+        $user_to_delete = User::withTrashed()->find(request()->route('user'));
+        $messages = [];
 
         if ($user_to_delete) {
 
             $messages = array_merge([
 
+                'user.exists' => 'Boop. '.trans('admin/users/message.user_not_found'),
```

Comment:
I think this might be more testing text.

Reply:
That was already removed